Assets/Vimeo/Scripts/Player/VimeoPlayer.cs

```diff
@@ -195,42 +198,40 @@ public void Play()
         {
             if (!IsVideoMetadataLoaded()) {
                 LoadAndPlayVideo();
-            }
+            } 
             else if (!videoControllerReady) {
-                VideoControllerPlayVideo();
-            }
+                StartCoroutine(VideoControllerPlayVideo());
+            } 
             else {
                 controller.Play();
             }
         }
 
-        private void VideoControllerPlayVideo()
+        private IEnumerator VideoControllerPlayVideo()
```

Comment:
this is a breaking change. You will need to make sure all paths yield a return. When you dont have AVPro installed you get errors.

Reply:
https://github.com/vimeo/vimeo-unity-sdk/pull/77/commits/acdb4a51a811535cf928d8cbcfd1e8ea031fcf14